packages/fxa-content-server/server/lib/routes/react-app/route-definition-index.js

```diff
@@ -55,6 +55,9 @@ function getIndexRouteDefinition(config) {
   const FEATURE_FLAGS_RECOVERY_CODE_SETUP_ON_SYNC_SIGN_IN = config.get(
     'featureFlags.recoveryCodeSetupOnSyncSignIn'
   );
+  const FEATURE_FLAGS_RECOVERY_PHONE_RESET_PASSWORD_2FA = config.get(
```

Comment:
Should this match the variable name in configuration.js? There it is `FEATURE_FLAGS_RECOVERY_PHONE_PASSWORD_RESET_2FA`